### PR TITLE
Fix strings, mostly old "Satellite" ocurrences

### PR DESCRIPTION
**Pull request by paususe**
## What does this PR change?

Fixes some uses of "Satellite", and others

- 'proxy' refers to SUSE Manager Proxy not to some random HTTP proxy, or any other kind of proxy. Make that explicit.
- Uppercase where needed
- Use PRODUCT_NAME and VENDOR_NAME intead of SUSE Manager and SUSE, and replace Satellite with PRODUCT_NAME

## GUI diff

This will impact translations.

Before:

After:

- [ ] **DONE**

## Documentation
- No documentation needed: **add explanation. This can't be used if there is a GUI diff**
- Documentation issue was created: [Link for SUSE Manager contributors](https://github.com/SUSE/spacewalk/issues/new?template=ISSUE_TEMPLATE_DOCUMENTATION.md&labels=documentation&projects=SUSE/spacewalk/31), [Link for community contributors](https://github.com/uyuni-project/uyuni-docs/issues/new).
- (OPTIONAL) [Documentation PR](https://github.com/uyuni-project/uyuni-docs/pulls)

- [ ] **DONE**

## Test coverage
- No tests: **add explanation**
- Unit tests were added
- Cucumber tests were added

- [ ] **DONE**

## Links

Fixes #
Tracks # **add downstream PR, if any**

- [ ] **DONE**

## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_lint_checkstyle"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_oracle"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

File: java/spacewalk-java.changes [hunk 1]
```diff
@@ -23,6 +23,7 @@
 - Add virtual network Start, Stop and Delete actions
 - Add virtual network list page
 - update default product tree tag and set Beta tag again
+- Fix strings (mentions of Satellite, replace SUSE Manager with PRODUCT_NAME, etc)
```
Comment (mcalmer) [inline java/spacewalk-java.changes:26]: new changelog entries on top please.

File: susemanager/susemanager.changes [hunk 2]
```diff
@@ -1,6 +1,7 @@
 - support installer update channels during autoinstallation
 - migrate all occurrences of kickstart to autoinstall in cobbler database (bsc#1169780)
 - define bootstrap repo data for SUSE Manager Proxies (bsc#1174470)
+- Fix strings (mentions of Satellite, replace SUSE Manager with PRODUCT_NAME, etc)
```
Comment (mcalmer) [inline susemanager/susemanager.changes:4]: new changelog entries on top please.

File: backend/spacewalk-backend.changes [hunk 3]
```diff
@@ -3,6 +3,7 @@
 - Take care of SCC auth tokens on DEB repos GPG checks (bsc#1175485)
 - Use spacewalk keyring for GPG checks on DEB repos (bsc#1175485)
 - Remove duplicate languages and update translation strings
+- Fix strings (mentions of Satellite, replace SUSE Manager with PRODUCT_NAME, etc)
```
Comment (mcalmer) [inline backend/spacewalk-backend.changes:6]: new changelog entries on top please.

**Review by mcalmer — APPROVED** [review]
Looks good so far. Just the changelog entries should go on top.

**Review by parlt91 — APPROVED** [review]
Other than what Micheal already pointed out.
LGTM :+1:

**Pull request merged**